sdk/appconfiguration/app-configuration/README.md

```diff
@@ -175,6 +178,134 @@ async function run() {
 run().catch((err) => console.log("ERROR:", err));
 ```
 
+### Create and get a snapshot
+
+```javascript
+const appConfig = require("@azure/app-configuration");
+
+const client = new appConfig.AppConfigurationClient(
+  "<App Configuration connection string goes here>"
+);
+
+const key = "testkey";
+const value = "testvalue";
+const label = "optional-label";
+
+const newSetting = await client.addConfigurationSetting({
+  key,
+  value,
+  label
+});
+const newSnapshot = await client.addSnapshot("testsnapshot", {
```

Comment:
```suggestion
const newSnapshot = await client.beginCreateSnapshotAndWait("testsnapshot", {
```

Also, mention the standard way, `await client.beginCreateSnapshot()` to get the poller.